pandora-tests/test/utils/utils.ts

```diff
@@ -0,0 +1,25 @@
+export function Assert(condition: unknown, msg?: string): asserts condition {
+	if (!condition) {
+		throw new Error(msg ? `Assetion failed: ${msg}` : 'Assertion failed');
+	}
+}
+
+/**
+ * Assert all arguments are `never`
+ *
+ * Useful for checking all possible outcomes are handled
+ */
+export function AssertNever(...args: never[]): never {
```

Comment:
wouldn't it be better to use these from common, or what is the reason not to import that?